scripts/local_testnet/start_local_testnet.sh

```diff
@@ -92,12 +97,6 @@ execute_command_add_PID() {
     echo "$!" >> $PID_FILE
 }
 
-# Start ganache, setup things up and start the bootnode.
-# The delays are necessary, hopefully there is a better way :(
-
-# Delay to let ganache to get started
-execute_command_add_PID ganache_test_node.log ./ganache_test_node.sh
-sleeping 10
 
 # Setup data
 echo "executing: ./setup.sh >> $LOG_DIR/setup.log"
```

Comment:
probably want to pass `$genesis_file` to the `setup.sh` file and use it there (currently hard coded to genesis.json)

Reply:
./setup.sh $genesis_file >> $LOG_DIR/setup.log 2>&1

Reply:
I moved the genesis time change to `start_local_testnet` script. Would be nice to keep all the genesis file logic contained to one file.